java/configuration/src/main/java/sleeper/configuration/Utils.java

```diff
@@ -50,6 +50,10 @@ public static boolean isPositiveInteger(String integer) {
         return parseAndCheckInteger(integer, num -> num > 0);
     }
 
+    public static boolean isPositiveIntegerOrNull(String integer) {
+        return (isPositiveInteger(integer)) || (integer == null);
```

Comment:
With the null check second, it'll only check if the string is null after it's called `Integer.parse` and found that it's invalid. We can put the null check first to avoid that. It might also be easier to read without the extra brackets?